web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -190,6 +190,10 @@ class ComboMarkdownEditor {
     // EasyMDE's CSS should be loaded via webpack config, otherwise our own styles can not overwrite the default styles.
     const {default: EasyMDE} = await import(/* webpackChunkName: "easymde" */'easymde');
     const easyMDEOpt = {
+      autosave: {
+        enabled: true,
+        uniqueId: this.textarea.id
```

Comment:
Thanks for this PR! Is this ID unique to the textarea on issue pages as a whole, or does the textarea on each issue page have its own ID? The context on why I am asking is that users may have "drafts" for many issue comments, and if it were the same ID for all issue pages then they would only be able to have one draft.

A followup on @silverwind's comment re: removal of easymde, is we could use https://github.com/github/session-resume to achieve the same behaviour with the new text editor.

Reply:
https://github.com/github/session-resume looks interesting and can be used in place of `jquery.are-you-sure` for automatic form restoration.

Reply:
Actually `session-resume` doesn't work well either. Think about 2 cases:

1. Content in the textarea -> submit the form -> clean the draft -> network fails -> content lost
2. Content in the textarea -> submit the form by AJAX -> draft is still in local storage

That's why "AJAX" is before the "LocalStorage Draft" in my summary issue #23290